share/smack/lib/smack.c

```diff
@@ -1006,6 +1008,25 @@ void __SMACK_decls() {
 #undef D
 }
 
+//The size parameter represents number of bits that are being accessed.
```

Comment:
@zvonimir the function getTypeSizeInBits in class DataLayout only provides size in bits for any type.